blscurve/bls_sig_min_pubkey.nim

```diff
@@ -256,3 +256,18 @@ func fastAggregateVerify*[T: byte|char](
   if not aggAffine.aggregateAll(publicKeys):
     return false
   return coreVerifyNoGroupCheck(aggAffine, message, signature, DST)
+
+func fastAggregateVerify*[T: byte|char](
```

Comment:
I think in general, we should be getting rid of these pre-composed helpers that add surface without adding significant functionality..